sphinx/domains/python.py

```diff
@@ -62,6 +63,34 @@
 }
 
 
+def _parse_arglist(arglist: str) -> addnodes.desc_parameterlist:
+    """Parse a list of arguments using python parser"""
+    params = addnodes.desc_parameterlist(arglist)
+    sig = signature_from_str('(%s)' % arglist)
+    for param in sig.parameters.values():
+        node = addnodes.desc_parameter()
+        if param.kind == param.VAR_POSITIONAL:
+            node += nodes.inline('', '*' + param.name, classes=['argument'])
+        elif param.kind == param.VAR_KEYWORD:
+            node += nodes.inline('', '**' + param.name, classes=['argument'])
+        else:
+            node += nodes.inline('', param.name, classes=['argument'])
```

Comment:
What about the `*` and `/` from signatures like the following? Does something insert those elsewhere?
```python
def foo(a, b, *, c=2): pass
def bar(a, b, /, **kwargs): pass
```

Reply:
The logic needs to be something like:
```
for param in params:
    if last_param_kind == POSITIONAL_ONLY and param.kind != POSITIONAL_ONLY :
        emit('/, ')
    elif last_param_kind == POSITIONAL_OR_KEYWORD and param.kind == KEYWORD_ONLY:
        emit('*, ')
    
    # ... as before

if last_param_kind == POSITIONAL_ONLY:
    emit('/')
```

Reply:
done